src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -24,8 +24,13 @@ export interface IIconLabelCreationOptions {
 	hoverDelegate?: IHoverDelegate;
 }
 
+export interface IIconLabelMarkdownString {
+	markdown: IMarkdownString | Promise<IMarkdownString | string | undefined> | string;
```

Comment:
Somehow I have a hard time to read this. Shouldn't this rather be written as:

```ts
markdown: IMarkdownString | string | undefined | Promise<IMarkdownString | string | undefined>;
```

So that both the promise result and the non-promise result are identical?

Reply:
Changed.